c2corg_api/models/route.py

```diff
@@ -322,3 +322,7 @@ class ArchiveRouteLocale(_RouteLocaleMixin, ArchiveDocumentLocale):
     'locales.title', 'locales.title_prefix', 'elevation_min', 'elevation_max',
     'activities'
 ])
+schema_association_waypoint_route = restrict_schema(schema_route, [
+    'locales.title', 'locales.title_prefix', 'elevation_min', 'elevation_max',
+    'activities', 'geometry.geom_detail'
```

Comment:
Most WP have less than 30 routes. And it would be really useful to visualize on the map the routes that go to / come from the current WP. That would help to figure out what route you're looking for when accessing the WP detail view.

For WP with many routes, you're probably right. But showing a subset of tracks on the map is consistent with showing a subset of cards in the associated routes section.